imports/server/setup.ts

```diff
@@ -120,23 +121,27 @@ Meteor.publish("googleScriptInfo", async function () {
   };
   const handle: Meteor.LiveQueryHandle = await cursor.observeAsync({
     added: (doc) => {
-      void (async () => {
+      (async () => {
         tracked = true;
         this.added(
           "googleScriptInfo",
           "googleScriptInfo",
           await formatDoc(doc),
         );
-      })();
+      })().catch((error) => {
+        Logger.error("googleScriptInfo added() failed", { error });
+      });
```

Comment:
(Similar question here about whether we should `this.error` and unwind the subscription on error)